homework/shelkovich_ilya/userValidationService/src/main/java/com/tutrit/java/quickstart/service/UserValidationService.java

```diff
@@ -0,0 +1,35 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.User;
+import com.tutrit.java.quickstart.exceptions.UserValidationException;
+
+public class UserValidationService {
+
+    public boolean validate(User user) throws UserValidationException {
+        try {
+            checkUserNameLength(user);
+        } catch (UserValidationException nameException) {
+            return false;
+        }
+        try {
+            checkUserAge(user);
+        } catch (UserValidationException ageException) {
+            return false;
+        }
+        return true;
```

Comment:
I think it wiil work. You can replase all code it methods on next cod:
```
checkUserNameLength(user);
checkUserAge(user);
return true;
```

Reply:
fixed